doc/source/development/contributing.rst

```diff
@@ -204,7 +204,8 @@ You will need `Build Tools for Visual Studio 2017
 	You DO NOT need to install Visual Studio 2019.
 	You only need "Build Tools for Visual Studio 2019" found by
 	scrolling down to "All downloads" -> "Tools for Visual Studio 2019".
-
+        In the installer, select the "C++ build tools" workload.
```

Comment:
Sorry for the nitpck, but can you remove the extra leading space in this line?

Reply:
Thanks @bashtage, I think I've fixed up the whitespace issue now and squashed the commit.